src/webos/install/weboswebruntime/data_locales.list

```diff
@@ -52,75 +52,53 @@ gen/content/app/strings/content_strings_uk.pak
 gen/content/app/strings/content_strings_vi.pak
 gen/content/app/strings/content_strings_zh-CN.pak
 gen/content/app/strings/content_strings_zh-TW.pak
-gen/neva/app_runtime/repack/af.pak
-gen/neva/app_runtime/repack/am.pak
-gen/neva/app_runtime/repack/ar.pak
-gen/neva/app_runtime/repack/as.pak
-gen/neva/app_runtime/repack/az.pak
-gen/neva/app_runtime/repack/bg.pak
-gen/neva/app_runtime/repack/bn.pak
-gen/neva/app_runtime/repack/bs.pak
-gen/neva/app_runtime/repack/cs.pak
-gen/neva/app_runtime/repack/da.pak
-gen/neva/app_runtime/repack/de.pak
-gen/neva/app_runtime/repack/el.pak
-gen/neva/app_runtime/repack/en-GB.pak
-gen/neva/app_runtime/repack/en-US.pak
-gen/neva/app_runtime/repack/es-419.pak
-gen/neva/app_runtime/repack/es.pak
-gen/neva/app_runtime/repack/et.pak
-gen/neva/app_runtime/repack/eu.pak
-gen/neva/app_runtime/repack/fake-bidi.pak
-gen/neva/app_runtime/repack/fa.pak
-gen/neva/app_runtime/repack/fi.pak
-gen/neva/app_runtime/repack/fr-CA.pak
-gen/neva/app_runtime/repack/fr.pak
-gen/neva/app_runtime/repack/ga.pak
-gen/neva/app_runtime/repack/gu.pak
-gen/neva/app_runtime/repack/ha.pak
-gen/neva/app_runtime/repack/he.pak
-gen/neva/app_runtime/repack/hi.pak
-gen/neva/app_runtime/repack/hr.pak
-gen/neva/app_runtime/repack/hu.pak
-gen/neva/app_runtime/repack/id.pak
-gen/neva/app_runtime/repack/it.pak
-gen/neva/app_runtime/repack/ja.pak
-gen/neva/app_runtime/repack/kk.pak
-gen/neva/app_runtime/repack/km.pak
-gen/neva/app_runtime/repack/kn.pak
-gen/neva/app_runtime/repack/ko.pak
-gen/neva/app_runtime/repack/ku.pak
-gen/neva/app_runtime/repack/lt.pak
-gen/neva/app_runtime/repack/lv.pak
-gen/neva/app_runtime/repack/mk.pak
-gen/neva/app_runtime/repack/ml.pak
-gen/neva/app_runtime/repack/mn.pak
-gen/neva/app_runtime/repack/mr.pak
-gen/neva/app_runtime/repack/ms.pak
-gen/neva/app_runtime/repack/nb.pak
-gen/neva/app_runtime/repack/nl.pak
-gen/neva/app_runtime/repack/or.pak
-gen/neva/app_runtime/repack/pa.pak
-gen/neva/app_runtime/repack/pl.pak
-gen/neva/app_runtime/repack/ps.pak
-gen/neva/app_runtime/repack/pt-BR.pak
-gen/neva/app_runtime/repack/pt-PT.pak
-gen/neva/app_runtime/repack/ro.pak
-gen/neva/app_runtime/repack/ru.pak
-gen/neva/app_runtime/repack/sk.pak
-gen/neva/app_runtime/repack/sl.pak
-gen/neva/app_runtime/repack/sq.pak
-gen/neva/app_runtime/repack/sr.pak
-gen/neva/app_runtime/repack/sv.pak
-gen/neva/app_runtime/repack/ta.pak
-gen/neva/app_runtime/repack/te.pak
-gen/neva/app_runtime/repack/th.pak
-gen/neva/app_runtime/repack/tr.pak
-gen/neva/app_runtime/repack/uk.pak
-gen/neva/app_runtime/repack/ur.pak
-gen/neva/app_runtime/repack/uz.pak
-gen/neva/app_runtime/repack/vi.pak
-gen/neva/app_runtime/repack/zh-CN.pak
-gen/neva/app_runtime/repack/zh-HK.pak
-gen/neva/app_runtime/repack/zh-TW.pak
-gen/neva/app_runtime/repack/zu.pak
+locales/am.pak
```

Comment:
Do not drop the gen/paths. About locales... we are already installing htem in the /opt/chromium68/locales path. At least for chrome specific paths we can keep that as it is now.

Reply:
done.